src/test/java/dasniko/testcontainers/keycloak/KeycloakContainerHttpsTest.java

```diff
@@ -69,6 +73,17 @@ public void shouldThrowNullPointerExceptionUponNullTlsKeystore() {
         assertThrows(NullPointerException.class, () -> new KeycloakContainer().useTlsKeystore("keycloak.jks", null));
     }
 
+    @Test
+    public void shouldAdminClientBeAbleToConnect() {
+        try (KeycloakContainer keycloak = new KeycloakContainer().useTls()) {
+            keycloak.start();
+
+            Keycloak admin = keycloak.getKeycloakAdminClient();
+            ServerInfoResource serverInfoResource = admin.serverInfo();
+            assertNotNull(serverInfoResource.getInfo());
+        }
+    }
```

Comment:
Good first test case. Please add another two tests, one for `.useTls(tlsCertificateFilename, tlsCertificateKeyFilename)` and one for `.useTlsKeystore(tlsKeystoreFilename, tlsKeystorePassword)` like the already existing TLS basic tests are available.
In a quick manual test, at least the `.useTls(String, String)` test failed, so there might be additional work to be done.

Reply:
Indeed, the certificate case was not in my personal need.
I'll fix it too.